### ISSUE TITLE
Explicit Syntax Highlighting Support

### ISSUE THREAD

**Issue by michaeltlombardi** [justinwalsh/daux.io#412]
Right now, Daux seems to be able to detect (reasonably well) what language an indented code block is and use `highlight.js` to give us syntax highlighting.

Unfortunately, it doesn't seem to work well with PowerShell (which is the majority of code blocks I am testing today). Historically, we've used code fences and explicitly named the language at the top of the fence, as in the example below.

``````
```someLanguage
# a Comment
Something(parameter)
```
``````

This works with GitHub Flavored Markdown and is used by several other mardown-syntax-highlighters.

Is this possible to configure with Daux? I've been going through the documentation today to do up a POC for our team.

**Comment by onigoetz** [comment]
Hello,
currently, adding the language to the code fenced block works as you explained.

However, our current build of highlight.js doesn't support powershell.
I updated to the latest version and added all languages highlight.js is able to support.

You can download the latest version on the  `development` branch and use powershell highlight

```
composer global require justinwalsh/daux.io:dev-development
```

**Comment by michaeltlombardi** [comment]
Thank you for the crazy fast turn-around! :)

**Comment by onigoetz** [comment]
This is now fixed and released in `0.3.0`

**Comment by michaeltlombardi** [comment]
Thank you!
